content/02.intro.md

```diff
@@ -1,24 +1,24 @@
+Automated machine learning (AutoML) systems were developed to automate this challenging and time-consuming process.
+These intelligent systems increase the accessibility and scalability of various machine learning applications by efficiently solving an optimization problem to discover pipelines that yield satisfactory outcomes, such as prediction accuracy.
 Consequently, AutoML allows data scientists to focus their effort in applying their expertise in other important research components such as developing meaningful hypotheses or communicating the results.
 
-[other AutoML systems]
+To our knowledge, there are several AutoML systems: Both auto-sklearn [@url:https://papers.nips.cc/paper/5872-efficient-and-robust-automated-machine-learning] and Auto-WEKA [@arxiv:1208.3719, @url:http://www.jmlr.org/papers/v18/16-261.html] uses Bayesian optimization for machine learning model/pipeline selection and hyperparameter optimization. Both Recipe [@doi:10.1007/978-3-319-55696-3_16] and autostacker[@arxiv:1803.00684] automates hyperparameter tuning and model selection using evaluation algorithm. Recipe is pipeline optimization through grammar-based genetic programming using grammars to define pipeline structure, while autostacker automates stacked ensembling. And DEvol [@url:https://github.com/joeddav/devol] automates deep neural network design specifically via genetic programming. H2O.ai [@url:http://docs.h2o.ai/h2o/latest-stable/h2o-docs/automl.html] automates data preprocessing, hyperparameter tuning, random grid search and stacked ensembles in a distributed ML platform in multiple languages. And Xcessiv [@url:https://github.com/reiinakano/xcessiv] provides web-based application for quick, scalable, and automated hyper-parameter tuning and stacked ensembling in Python.
 
-Tree-based Pipeline Optimization Tool (TPOT) is a genetic programming-based AutoML system that automates the laborious process of designing a machine learning pipeline to solve a supervised learning problem. 
-At its core, TPOT uses genetic programming (GP) [@raw:GPbook] to optimize a series of feature preprocessors and machine learning models with the objective of maximizing classification accuracy. 
-While most AutoML systems primarily focus on model selection and hyperparameter optimization, TPOT also pays attention to feature selection and feature engineering in building a complete pipeline. Applying GP with the NSGA-II Pareto optimization [@doi:10.1109/4235.996017], TPOT optimizes the accuracy achieved by the pipeline while accounting for its complexity. 
+Tree-based Pipeline Optimization Tool (TPOT) is AutoML system using strongly typed genetic programming and it automates the laborious process of designing a machine learning pipeline to solve a supervised learning problem.
```

Comment:
We will probably need to discuss this change in person. I thought strongly typed was only when we include Template?

Reply:
TPOT was strongly typed GP in the 1st paper of TPOT.